README.md

```diff
@@ -1,4 +1,5 @@
-[![Appveyor build status (Windows)](https://ci.appveyor.com/api/projects/status/xfllw9vkp3lj4l0v/branch/main?svg=true)](https://ci.appveyor.com/project/ispc/ispc/branch/main)
+[![Appveyor build status (Windows)](https://ci.appveyor.com/api/projects/status/xfllw9vkp3lj4l0v/branch/main?svg=true)](https://ci.appveyor.com/project/ispc/ispc/branch/main) [![ispc](https://snapcraft.io/ispc/badge.svg)](https://snapcraft.io/ispc)
```

Comment:
Is the badge for `1.19.0`? If so, then it's confusing to have its badge next to main build status.

I guess we should have a link to it on the release page for 1.20.0 when we release.

Reply:
Yes, it is. OK.